src/Sylius/Behat/Context/Ui/Shop/CartContext.php

```diff
@@ -303,10 +303,10 @@ public function iAddQuantityOfProductsToTheCart($quantity, ProductInterface $pro
     }
 
     /**
-     * @Given I have :quantity products :product in the cart
-     * @When I add :quantity products :product to the cart
+     * @Given /^I have(?:| added) (\d+) (products "([^"]+)") (?:to|in) the cart$/
```

Comment:
Please remove inner brackets for transformer: `(products "([^"]+)")` -> ` (products "[^"]+")`

Same below.